googleSchema.js

```diff
@@ -0,0 +1,55 @@
+// Philosophy FAQ
+// to add more philosophy questions just add them in this json as the first 2 and the script will do the rest
+var faqPhilosophy = {
+	"faqs": {
+		"protectingOurDemographics": {
+			"question": "What does Destiny think about protecting our demographics?",
+			"answer": "Destiny believes that this endeavor is futile because he rejects the notion that there is a consistent and coherent definition of American values",
+		},
+		"protestingRioting": {
+			"question": "What does Destiny think about the protesting and rioting?",
+			"answer": "Destiny ardently believes in people's right to protest and will always support the right to protest, even if he doesn't necessarily agree with the positions being advocated for by any particular protest. ",
+		},
+	}
+};
+
+// Initialize JSON array and data
+var faqsPhilosophy = faqPhilosophy.faqs;
+faqsPhilosophy = JSON.stringify(faqsPhilosophy);
+faqsPhilosophy = JSON.parse(faqsPhilosophy);
+
+var string = `{"@context": "https://schema.org","@type": "FAQPage","mainEntity":[`;
```

Comment:
You already using template string so you should take advantage of this. Line 21 to line 33 can be in a single template string.

Line 25 to line 30 can be simplified with the Array.join fonction